### PR TITLE
Bump UI to v2.7.1

### PR DESCRIPTION
**Pull request by feedmeapples**
**What changed?**
Bump UI to v2.7.1. 

**Why?**
Keep dependencies up to date.

**How did you test it?**
N/A

**Potential risks**
No risks.

**Is hotfix candidate?**
No.

### CODE REVIEW

**Comment by mindaugasrukas** [comment]
Can we bump it to 2.7.1?

File: develop/docker-compose/docker-compose.cdc.yml [hunk 1]
```diff
@@ -5,12 +5,12 @@ version: "3.5"
 
 services:
   temporal-ui-standby:
-    image: temporalio/ui:2.2.3
+    image: temporalio/ui:2.7.0
```
Comment (feedmeapples) [inline develop/docker-compose/docker-compose.cdc.yml:8]:
```suggestion
    image: temporalio/ui:2.7.1
```

File: develop/docker-compose/docker-compose.cdc.yml [hunk 2]
```diff
@@ -5,12 +5,12 @@ version: "3.5"
...
 services:
   temporal-ui-standby:
-    image: temporalio/ui:2.2.3
+    image: temporalio/ui:2.7.1
     container_name: temporal-dev-ui-standby
     environment:
       - TEMPORAL_UI_PORT=8081
   temporal-ui-other:
-    image: temporalio/ui:2.2.3
+    image: temporalio/ui:2.7.0
```
Comment (feedmeapples) [inline develop/docker-compose/docker-compose.cdc.yml:13]:
```suggestion
    image: temporalio/ui:2.7.1
```

File: develop/docker-compose/docker-compose.yml [hunk 3]
```diff
@@ -79,7 +79,7 @@ services:
     image: temporalio/web:1.15.0
     container_name: temporal-dev-web
   temporal-ui:
-    image: temporalio/ui:2.2.3
+    image: temporalio/ui:2.7.0
```
Comment (feedmeapples) [inline develop/docker-compose/docker-compose.yml:82]:
```suggestion
    image: temporalio/ui:2.7.1
```

**Comment by feedmeapples** [comment]
> Can we bump it to 2.7.1?

done

**Review by mindaugasrukas — APPROVED** [review]

**Review by alexshtin — APPROVED** [review]

**Pull request merged**